.yarnrc.yml

```diff
@@ -1,9 +1,7 @@
-nodeLinker: node-modules
+compressionLevel: mixed
```

Comment:
The `yarn set version latest` command made these changes.

Interestingly, the default for `compressionLevel` is 0, not mixed, per https://yarnpkg.com/configuration/yarnrc#compressionLevel 🤔